packages/pwa-kit-dev/package.json

```diff
@@ -1,127 +1,127 @@
+        "@babel/plugin-transform-modules-commonjs": "^7.21.2",
+        "@babel/plugin-transform-object-assign": "^7.18.6",
+        "@babel/plugin-transform-runtime": "^7.21.0",
+        "@babel/preset-env": "^7.20.2",
+        "@babel/preset-react": "^7.18.6",
+        "@babel/preset-typescript": "^7.21.0",
+        "@babel/register": "^7.21.0",
+        "@babel/runtime": "^7.21.0",
+        "@babel/runtime-corejs2": "^7.21.0",
+        "@babel/traverse": "^7.21.3",
+        "@loadable/babel-plugin": "^5.15.3",
+        "@loadable/server": "^5.15.3",
+        "@loadable/webpack-plugin": "^5.15.2",
+        "@pmmmwh/react-refresh-webpack-plugin": "^0.5.10",
+        "archiver": "1.3.0",
+        "babel-jest": "^26.6.3",
+        "babel-loader": "^8.3.0",
+        "babel-plugin-dynamic-import-node-babel-7": "^2.0.7",
+        "babel-plugin-formatjs": "10.4.0",
+        "chalk": "^4.1.2",
+        "commander": "^9.5.0",
+        "compression": "1.7.4",
+        "copy-webpack-plugin": "^9.1.0",
+        "cross-env": "^5.2.1",
+        "eslint": "7.32.0",
+        "eslint-config-prettier": "4.3.0",
+        "eslint-import-resolver-webpack": "0.13.2",
+        "eslint-plugin-header": "^3.1.1",
+        "eslint-plugin-import": "2.27.5",
+        "eslint-plugin-jsx-a11y": "6.7.1",
+        "eslint-plugin-prettier": "3.4.1",
+        "eslint-plugin-react": "^7.32.2",
+        "express": "^4.18.2",
+        "fs-extra": "^10.1.0",
+        "git-rev-sync": "^3.0.2",
+        "glob": "7.2.3",
+        "ignore-loader": "^0.1.2",
+        "jest": "^26.6.3",
+        "jest-cli": "^26.6.3",
+        "jest-environment-jsdom": "^26.6.2",
+        "jest-environment-jsdom-global": "^2.0.4",
+        "jest-expect-message": "1.1.3",
+        "jest-fetch-mock": "^2.1.2",
+        "mime-types": "2.1.35",
+        "minimatch": "3.1.2",
+        "open": "^8.4.2",
+        "prettier": "^2.8.6",
+        "pwa-kit-runtime": "2.8.0-dev",
+        "react-refresh": "^0.14.0",
+        "replace-in-file": "^6.3.5",
+        "request": "^2.88.0",
+        "require-from-string": "^2.0.2",
+        "rimraf": "2.7.1",
+        "speed-measure-webpack-plugin": "^1.5.0",
+        "svg-sprite-loader": "^6.0.11",
+        "validator": "^13.9.0",
+        "watch": "1.0.2",
+        "webpack": "^5.76.3",
+        "webpack-bundle-analyzer": "^4.8.0",
+        "webpack-cli": "^4.10.0",
+        "webpack-dev-middleware": "^5.3.3",
+        "webpack-hot-middleware": "^2.25.3",
+        "webpack-hot-server-middleware": "^0.6.1",
+        "webpack-notifier": "^1.15.0",
+        "ws": "^8.13.0"
+    },
+    "devDependencies": {
+        "@loadable/component": "^5.15.3",
+        "internal-lib-build": "2.8.0-dev",
+        "nock": "^13.3.0",
+        "superagent": "^6.1.0",
+        "supertest": "^4.0.2"
+    },
+    "peerDependencies": {
+        "@loadable/component": "^5.15.0"
+    },
+    "engines": {
+        "node": "^16.0.0 || ^18.0.0",
+        "npm": "^7.0.0 || ^8.0.0 || ^9.0.0"
+    },
```

Comment:
Should we move this up the file? Just for consistency

Reply:
lol, you're not wrong... but let's do it in some unrelated PR so I don't have to re-run CI! 😓